code/game/objects/items.dm

```diff
@@ -930,6 +930,7 @@
 //Called when the item blocks an attack. Return 1 to stop the hit, return 0 to let the hit go through
 /obj/item/proc/on_block(damage, atom/blocked)
 	if(ismob(loc))
+	
```

Comment:
I think you forgot to code the part where this PR does what you said in the changelog?

Reply:
I swear I have it on my VS code. Whatever